code/datums/keybinding/xeno.dm

```diff
@@ -516,10 +516,4 @@
 	description = "Banish a creature or object a short distance away within line of sight to null space. Can target oneself and allies. Can be manually cancelled with Recall."
 	keybind_signal = COMSIG_XENOABILITY_BANISH
 
-/datum/keybinding/xeno/recall
-	name = "recall"
-	full_name = "Wraith: Recall"
-	description = "Recall a target from netherspace, ending Banish's effect."
-	keybind_signal = COMSIG_XENOABILITY_RECALL
```

Comment:
Pretty sure we can't remove keybinds. 
If this keybind is set and then removed from code, there will be a runtime everytime you user that key, with no possiblity of fixing it. 
Just let it there until we have a method to sanitize keybinds savefiles

Reply:
i remember there was a PR that allowed you to do so, we justnever tried to